src/nft/css/sprinkles.css.ts

```diff
@@ -7,6 +7,7 @@ const themeContractValues = {
     accentFailureSoft: '',
     accentAction: '',
     accentActionSoft: '',
+    accentActive: '',
```

Comment:
if it's the same in dark and light mode you can just add it to vars.color in sprinkles instead of to the themes